types/reactstrap/reactstrap-tests.tsx

```diff
@@ -4528,3 +4528,39 @@ function Example127() {
         </div>
     );
 }
+
+class Example128 extends React.Component<any, any> {
+  constructor(props: any) {
+    super(props);
+
+    this.toggle = this.toggle.bind(this);
+    this.state = {
+      dropdownOpen: false,
+    };
+  }
+
+  toggle() {
+    this.setState({
+      dropdownOpen: !this.state.dropdownOpen,
+    });
+  }
+
+  render() {
+    return (
+      <Dropdown
+        isOpen={this.state.dropdownOpen}
+        toggle={this.toggle}
+        positionFixed
```

Comment:
positionFixed should be on the DropdownMenu element

Reply:
Oh... thanks. I will fix it 👍